### PR TITLE
feature: Add first step requirements

### PR DESCRIPTION
**Pull request by nevisende**
- I used Gitflow.
- I created an npm project with webpack. 
- I wrote plain HTML markup with minimum styling.I used this wireframe as a guide:
- <p align="center" dir="auto">
  <a target="_blank" rel="noopener noreferrer" href="/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png">Basic UI</a>
</p>
- I didn't make API calls.
- I used ES6 modules.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @nevisende 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline webpack.config.js:1]: - [ ] Kindly remove this eslint disable line and fix the issue in the code.

File: dist/486.bundle.js.LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
+ * @license
+ * Lodash <https://lodash.com/>
+ * Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
```
Comment (mikenath223) [inline dist/486.bundle.js.LICENSE.txt:4]: - [ ] Please update the readme.md following [this template](https://github.com/microverseinc/readme-template).

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @nevisende 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] For this project, you are to use gitflow. I can see that you made the development branch your default branch, that is the first step to using gitflow. The next step would be to create a feature branch from development branch. This means when you create a pull request, it would be to merge the feature branch into the development branch and not the development branch into the main branch. I hope you understand, you can read more about [git-flow here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) or reach out to me if you need further clarification.

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

**Pull request merged**